gradle.properties

```
@@ -14,7 +14,7 @@ kotlin.js.ir.output.granularity=whole-program
 # Temporarily force IDEs to produce a Build Scan
 systemProp.org.gradle.internal.ide.scan=true
 # If you're experimenting with changes and don't want to update the verification file right away, please change the mode to "lenient" (not "off")
-org.gradle.dependency.verification=strict
+org.gradle.dependency.verification=lenient
```

plz how to fix this?

Did not worked out with: `./gradlew  --write-verification-metadata sha256 help`

```suggestion
org.gradle.dependency.verification=strict
```